ci/test-stable.sh

```diff
@@ -46,6 +46,16 @@ for program in programs/native/*; do
   )
 done
 
+# Run program/native/bpf_loader's test and bench with bpf_c feature
+(
+  set -x
+  cd "programs/native/bpf_loader"
+  echo --- program/native/bpf_loader test --features=bpf_c
```

Comment:
Why does bpf_loader even have a bpf_c feature flag in it at all?  When would we not want to build bpf_loader without C support?

Reply:
bpf_c feature builds the c programs, there are tests that don't require them.  If we want everyone to incur a llvm dependency then we can remove it and always build the bpf programs.  This can maybe go away when I introduce the llvm docker...

Reply:
Ok so `bpf_c` is really `bpf_c_tests`